docs/stencil-docs/developing-further/google-amp.md

```diff
@@ -119,7 +119,7 @@ If you are having any implementation issues, review the [full Pull Request #964]
 In versions 1.6.0+ of Cornerstone, there are a few key file locations where the AMP information is located:
 
 * Base AMP layout template is located in [templates/layout/amp.html](https://github.com/bigcommerce/cornerstone/blob/master/config.json).
-* The files for Google AMP are located in [/templates/pages/amp](https://github.com/bigcommerce/cornerstone/tree/master/templates/pages/amp). This is where `product.html` and `category.html` are located. (Note: there may be other files in this folder, but the product and category pages are the only one pages that currently support Google AMP).
+* The files for Google AMP are located in [/templates/pages/amp](https://github.com/bigcommerce/cornerstone/tree/master/templates/pages/amp). This is where `product.html` and `category.html` are located. (Note: There may be other files in this folder, but the product and category pages are the only pages that currently support Google AMP).

```

Comment:
Since we follow Chicago Manual of Style, we don't capitalize here:

>According to the Chicago Manual of Style, the first letter after a colon should be capitalized in the following cases:
> * The first word after the colon is a proper noun
> * The colon introduces 2 or more sentences
> * The colon introduces speech in dialogue, a quotation, or a question

[Sauce](https://www.tckpublishing.com/capitalization-after-colons/)

Reply:
@aglensmith, thanks for looking this over. I want to remind you and @traci707 to use the [Dev Docs Style Guide](https://intranet.bigcommerce.com/display/SST/Dev+Docs+Style+Guide) on Confluence. It's based on the Google style guide. Thanks!

Reply:
Sounds good, thank. you both.